clmm/support/mock_data.py

```diff
@@ -3,14 +3,20 @@
 import warnings
 from scipy import integrate
 from scipy.interpolate import interp1d
+from scipy.stats import chi2
```

Comment:
Is there any specific reason for importing this function? I did not find the use of this function in the rest of the file.

Reply:
you are right. I was testing ways to implement error distributions and forgot to remove it. Thanks for catching that!